gzip_test.go

```diff
@@ -306,21 +306,22 @@ func TestStatusCodes(t *testing.T) {
 	}
 }
 
-func TestStatusCodesFlushed(t *testing.T) {
+func TestFlushBeforeWrite(t *testing.T) {
+	b := []byte(testBody)
```

Comment:
You need to use the smallTestBody ?

Reply:
@Juliens Why's that? In this test case I'm testing the interaction between Flush and Write and I want the response to be gzipped.

Reply:
Sorry I misread, I believed the write was before the flush :wink: 

It's good for me even though I do not really like the 'Flush` after a small body writing does not do anything.